modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,95 @@
+
+* With this update, you can configure log levels such as "debug", "info", "warn", "error", "panic" and "fatal". The default log level set for the output is "info". Users can select an appropriate log level by adding the environment variable `LOG_LEVEL` in Red Hat OpenShift GitOps subscription's field, `spec.config.env`.
+
+.Example
+[source,yaml]
+----
+   yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: gitops-operator
+  namespace: openshift-gitops-operator
+spec:
+  ...
+  config:
+    env:
+    - name: LOG_LEVEL
+      value: "error"
+----
+
+* With this feature, Argo CD uses the wildcard values in `sourceNamespaces` to specify multiple namespaces or patterns of namespaces. To use this feature, specify the namespaces where Argo CD can manage applications in the Argo CD YAML with `spec.sourceNamespaces`. 
+
+.Example
+[source,yaml]
+----
+yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd-wildcard-pattern
+spec:
+  sourceNamespaces:
+    - app-team-*
+   - namespace-2
+----
+
+In this example, permissions are granted to namespaces that match the pattern `app-team-*`, such as `app-team-1`, `app-team-2`, and namespace-2, which does not use wildcard values.
+
+To grant permissions for all the namespaces on the Argo CD cluster using the `*` wildcard pattern, the following Argo CD CR configuration must be used:
+
+.Example
+[source,yaml]
+----
+yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd-all-namespaces
+spec:
+  sourceNamespaces:
+    - '*'
+----
+
+[id="fixed-issues-1-12-0_{context}"]
+== Fixed issues
+
+* Before this update, the notifications controller could not use `built-in` functions in notification templates to obtain information about applications due to an incorrect repo-server address. This update fixes the issue by initializing the notification controller to use the correct repo-server address.
+
+* In this update, users can go to the Argo CD Web UI and create job executions from a `cron` job defined in a deployed Argo CD application.
+
+* In this update, the application environment card gets removed automatically after the Argo CD application is deleted. Refreshing or reloading of the Argo CD Web UI does not remove the application environment card.
+
+* Before this update, when a user deletes apps and namespaces in Argo CD, in some cases, the Red Hat GitOps plugin displays an error message. This update fixes the issue by updating some components from the dynamic plugin SDK and provides better error handling. As a result, these error messages are not displayed after deleting an apps or namespaces.
+
+* Before this update, the password for an admin user was reset to the default password automatically when the Argo CD CLI or Web UI was updated. This update fixes the issue and the user can now update the admin password and the password is no longer set to default.
```

Comment:
The original text was "password for admin user used to reset with default password when updated by ArgoCD CLI or UI"... it's a nuanced difference but the password being updated by the CLI/UI vs when the CLI/UI are updated are different